webapi/src/test/scala/org/knora/webapi/responders/v2/ResourcesResponseCheckerV2.scala

```diff
@@ -42,6 +44,10 @@ object ResourcesResponseCheckerV2 {
                 assert(expectedResource.label == receivedResource.label, "label does not match")
                 assert(expectedResource.resourceClass == receivedResource.resourceClass, "resource class does not match")
 
+                // this check is necessary because zip returns a sequence of the length of the smaller of the two lists to be combined.
```

Comment:
@benjamingeer I have just noticed that this method never really checked if the expected values matched the received values. If you expect some values but you do not receive any (empty sequence of values), zip returns an empty collection, so no checks are carried out ...

Reply:
That's great. :)

Reply:
Do we need tests for our tests (meta tests)? And if so, where would it end?

Reply:
I already made a `ResponseCheckerR2RV2Spec`, you can add a test there for this case. :)

Reply:
done in https://github.com/dhlab-basel/Knora/pull/862/commits/72e76b8885ec91442fb9d11e9596be962a8c06e8

Reply:
and an additional check for v1